Devscord.DiscordFramework/Framework/Commands/Responses/Resources/Responses.resx

```diff
@@ -132,14 +132,17 @@
   <data name="AvailableSafeRoles" xml:space="preserve">
     <value>Dostępne role: {{roles}}</value>
   </data>
-  <data name="ChannelNotFound" xml:space="preserve">
-    <value>Nie znaleziono kanału {{channel}} </value>
+  <data name="EmptyArgument" xml:space="preserve">
+    <value>Argument {{argument}} jest nie może być pusty.</value>
+  </data>
+  <data name="GetUserWarns" xml:space="preserve">
+    <value>Ostrzeżenia użytkownika {{user}}: {{warns}}</value>
   </data>
   <data name="InvalidArguments" xml:space="preserve">
-    <value>Podane argumenty są niepoprawne</value>
+    <value>Dla tego polecenia są dostępne tylko argumenty: {{arguments}}</value>
```

Comment:
brakuje kropki na końcu

Reply:
to też z git pulla, dodane

Reply:
hmm, na masterze jest ta wersja `Podane argumenty są niepoprawne` i taka powinna zostać
wgl widzę że responses doszły stare, a na masterze są inne
coś mi się wydaje że źle rozwiązałeś konflikty

Reply:
ok teraz zaciągnąłem z mastera i dokleiłem swoje